File: c1/p4/ans.cpp
#include <bits/stdc++.h>
using namespace std;

int n, odd, even;

signed main() {
    ios_base::sync_with_stdio(0);cin.tie(0);

    cin >> n;
    for (int i = 0, x; i < n; ++i) {
        cin >> x;
        odd += x % 2 == 1;
        even += x % 2 == 0;
    }
    cout << (odd == even ? "meow" :"...");

    return 0;
}

File: c1/p4/ans.py
# print((lambda n: "meow" if sum(1 for _ in range(n) if int(input()) % 2 == 0) == n // 2 else "...")(int(input())))

n = int(input())
treats = [int(input()) for _ in range(n)]
even, odd = 0, 0
for treat in treats:
    if treat % 2 == 0:
        even += 1
    else:
        odd += 1
if even == odd:
    print("meow")
else:
    print("...")